lem.sh

```diff
@@ -1,4 +1,4 @@
-#/bin/sh
+#!/bin/sh
 
 export LEMDIR=/usr/local/share/lem
-$LEMDIR/lem $*
+$LEMDIR/lem "$@"
```

Comment:
I don't think quotes are wanted here. This seems like a wrapper script, you should be able to call it with multiple arguments (and there multiple arguments shouldn't be passed as one argument to `lem`).

Reply:
Actually, the effect of `"$@"` is to remove spaces inside each argument. Not between them.

You can test it using this script:

```sh
#!/bin/sh

set -eu

mkdir "$@"
```

Then call it with: `./mkdir.sh -p d/d/d/d/d/d`. It'll work. :)

Reply:
Dang, your right! TIL.